evalml/tests/automl_tests/test_automl_algorithm.py

```diff
@@ -16,6 +16,9 @@ def next_batch(self):
             return self._dummy_pipelines.pop()
         raise StopIteration("No more pipelines!")
 
+    def _transform_parameters(self, pipeline, proposed_parameters):
```

Comment:
So I'm understanding correctly: this is overridden since we're using a dummy algo that would otherwise inherit our complicated impl?

Reply:
I made `_transform_parameters` an abstract method, so it needs to be implemented otherwise it'll throw an error. We are testing that in other places so I figured we can just have it be a pass here.